### PR TITLE
Allow Prettier in IDE for microservices

### PR DESCRIPTION
**Pull request by fraxachun**
https://github.com/vivid-planet/comet-starter/pull/95#issue-2064259483

Fixes the "bad IDE support" by allowing allowing prettier in IDE for api/admin/site/create-app but executing the cli-check only for root-files.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -5,7 +5,7 @@
         "dev": "dotenv -c secrets -- dev-pm start",
         "copy-schema-files": "node copy-schema-files.js",
         "lint": "run-p lint:*",
-        "lint:root": "npx prettier --check './**/*.{js,json,md,yml,yaml}'",
```
Comment (dkarnutsch) [inline package.json:8]: This does not work for values.yaml in chart/.

We would need to exclude admin/api/site/create-app manually.

**Review by dkarnutsch — CHANGES_REQUESTED** [review]

File: package.json [hunk 2]
```diff
@@ -5,7 +5,7 @@
...
+        "lint:root": "npx prettier --check './!(admin|api|site|create-app)/**/*.{js,json,md,yml,yaml}'",
```
Comment (nsams) [inline package.json:8]: now that this doesn't lint root alone anymore, maybe rename to:
```suggestion
        "lint:prettier": "npx prettier --check './!(admin|api|site|create-app)/**/*.{js,json,md,yml,yaml}'",
```

Comment (nsams) [reply package.json:8]: or, wait: des the `!` mean those will be excluded?

Comment (nsams) [reply package.json:8]: ok, so why this change? does it still fix "bad IDE support"?

**Review by nsams — APPROVED** [review]

**Review by johnnyomair — APPROVED** [review]
Finally, thanks!

**Comment by johnnyomair** [comment]
@dkarnutsch please review again, thanks!

**Review by dkarnutsch — APPROVED** [review]

**Review by thomasdax98 — APPROVED** [review]

**Pull request merged**